src/main/java/frc/robot/Robot.java

```diff
@@ -58,8 +63,7 @@ public void robotInit() {
         }
 
         Logger.start();
-
-        robotContainer = new RobotContainer();
+        RobotContainer robotContainer = new RobotContainer();
```

Comment:
Since `robotContainer` is already a private variable, we should make this line read:
```
robotContainer = new RobotContainer();
```